vignettes/KMs-Cycled-Per-Year.rmd

```diff
@@ -0,0 +1,199 @@
+---
+title: "Estimating million km's cycled per zone in England and Wales regions"
+author: "Nathanael Sheehan"
```

Comment:
---
title: "Estimating million km's cycled per zone in England and Wales regions
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{Reproducing cycling potential estimates in UK cities}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
author: 
- "Nathanael Sheehan"
---